docs/solutions/reference-designs/eval-cn0584-ebz/HDL-project/index.rst

```diff
@@ -0,0 +1,100 @@
+HDL Project
+============
+
+Introduction
+------------
+
+This page describes the HDL project for 
+:dokuwiki:`CN0584 Precision Low Latency Development Kit </resources/eval/user-guides/circuits-from-the-lab/cn0584>`
+and :dokuwiki:`CN0585 Quad Channel, Low Latency, Data Acquisition and Signal Generation Module </resources/eval/user-guides/circuits-from-the-lab/cn0585>`.
```

Comment:
@jvelasc2  these are referring to the wiki pages, these should point to the soon to be in existence GitHub docs pages.